modules/installation-osp-creating-image.adoc

```diff
@@ -49,7 +49,7 @@ $ openstack image create --container-format=bare --disk-format=qcow2 --file rhco
 ----
 +
 [IMPORTANT]
-Depending on your {rh-openstack} environment, you might be able to upload the image in either link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/15/html/instances_and_images_guide/index[`.raw` or `.qcow2` formats]. If you use Ceph, you must use the `.raw` format.
+Depending on your {rh-openstack} environment, you might be able to upload the image in either `.raw` or `.qcow2` formats. If you use Ceph, you must use the `.raw` format. You must follow link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/creating_and_managing_images/ch-image-service#section-convert-image[the manual process] to transform the image from `.qcow2` to `.raw`.
```

Comment:
This could be multiple lines. 

Not sure if "must" is required here. Is "Follow the manual process to transform the image from the `.qcow2` format to the `.raw`  format" sufficient?

Reply:
Addressed- it would be sufficient